### DISCUSSION TITLE
how to use component methods using composition api with typescript?

### DISCUSSION THREAD

**Discussion by wjndy4** [quasarframework/quasar#9606] [General - Components / Directives / etc] [answered]
Please tell me know how to use component methods using composition api with typescript. I googled for a long time but I can not find any example about that thing.
P/s: thanks anyone read this discussion. I found how to do it ^^

**Comment by metalsadman** [comment, marked as answer]
use a `ref`, then you can access that component's api ie. `someComponentRef.value.someComponentMethod(...)`. https://v3.vuejs.org/guide/composition-api-template-refs.html#template-refs

**Reply by wjndy4** [reply]
thank you for your reply ^^
